terraform/helm/fullnode/templates/fullnode.yaml

```diff
@@ -23,6 +23,7 @@ spec:
         seccomp.security.alpha.kubernetes.io/pod: runtime/default
         prometheus.io/scrape: "true"
         prometheus.io/port: "9101"
+        autoscaling.k8s.io/vertical-pod-autoscaler-enabled: "true"
```

Comment:
this would hardcode "true" as the option here and bypass the setting with the variable

Reply:
this is to enable vpa for statefulset, vpa implementation itself is wrapped by if condition of the variable, but yeah it's better to use variable here as well